tasks/go.py

```diff
@@ -307,9 +307,7 @@ def deps(
             print("set ANDROID_NDK_HOME to build android")
```

Comment:
We are not using `ndkhome` anymore, right? Can we remove this?

Reply:
I've done that on my other PR... If I remove it here it will conflict big time 😓  Let's leave it here for this PR?

Reply:
Sure, let's leave it here and do it on a new PR then